.github/pull_request_template.md

```diff
@@ -1,10 +1,17 @@
 Fixes/resolves #ISSUE_ID and fixes/resolves #OTHER_ISSUE_ID
 
-## Proposed Changes
+## 작업 배경
 - Fix something
 - Or Implement something
 
-## Notes
+## 작업 내용
+- Fix something
+- Or Implement something
+
+## 기타
```

Comment:
I'm not 100% sure if these changes should be done in this PR. I know there are language discrepancies, but I think it would be better to have a separate PR for translation.
이 내용이 이 PR에 있어야 하는지 잘 모르겠어요. 언어가 안맞는 문제가 있는건 알지만 차라리 새로운 PR에서 해결하는게 좋지 않을까 싶어요.

Reply:
제 생각엔 기존에 있던 Notes가 작업 내용은 아니지만 다른 사람에게 알려야 할 기타 내용들을 적는 곳이라고 생각해서
`기타`라는 항목을 만들었습니다.

작업 배경과 작업 내용에 적을 수는 없지만 꼭 적어야 할 내용들이 있지 않을까요

Reply:
I was talking about translation, not the ```기타``` section, since this PR's description isn't talking abouttranslation.
```기타``` 섹션 얘기가 아니라 번역 얘기였어요, PR 설명문에 번역 관련이 없어서 이 PR에 있는게 맞나 싶네요.

Reply:
내용이 잘 이해가 되질 않아서 유선으로 연락 드리겠습니다!

Reply:
내용 수정 후 푸쉬했습니다!